src/components/utils/blocks.ts

```diff
@@ -9,11 +9,18 @@ import { isFunction, isString, log } from '../utils';
  * We can merge two blocks if:
  *  - they have the same type
  *  - they have a merge function (.mergeable = true)
+ *  - If they have valid conversions config
  *
  * @param targetBlock - block to merge to
  * @param blockToMerge - block to merge from
  */
 export function areBlocksMergeable(targetBlock: Block, blockToMerge: Block): boolean {
+  if (blockToMerge.mergeable &&
```

Comment:
I'm not sure that `mergeable` is required for this case. If they are both providing correct `conversionConfig`, we can merge them without calling the `merge()` method

Reply:
It's done